src/loaders/secretsManager.js

```diff
@@ -0,0 +1,65 @@
+import AWS from 'aws-sdk';
+
+import { CREDSTASH_PREFIX } from '../constants';
+import { convert } from './convert';
+
+
+export function getClient() {
+    return new AWS.SecretsManager();
+}
+
+function convertAllKeys(obj) {
+    if (typeof obj !== 'object') return convert(obj);
```

Comment:
Not recursing into arrays?

Reply:
no

Reply:
This means that objects inside arrays are not going to be converted. Is this something we want?